### DISCUSSION TITLE
Why can't the logo be displayed

### DISCUSSION THREAD

**Discussion by myuanzhang** [squidfunk/mkdocs-material#3700] [Q&A] [answered]
I don't know why the logo I configured is not displayed.
![mkdocs](https://user-images.githubusercontent.com/68680964/158020251-78e68283-c053-42dc-b038-efa68ac85f8e.png)

Here is my configuration:
![mkdocs2](https://user-images.githubusercontent.com/68680964/158020326-858f84d8-6afa-4b67-a46a-381b207ce628.png)

I have put the logo in the corresponding folder
![Image 4](https://user-images.githubusercontent.com/68680964/158020530-2a5cf86a-8fcb-4401-8b50-3dfc7284605c.png)

**Comment by lampensau** [comment, marked as answer]
It seems like your `logo:` configuration in your mkdocs.yml is pointing to a different path. Your configuration expects the `logo.png` to reside in `E:\mkdocs\my-project\docs\assets\`.

**Reply by myuanzhang** [reply]
Oh, I got it. Thank you very much!
